Userland/Libraries/LibJS/Runtime/ArrayBuffer.h

```diff
@@ -102,7 +102,7 @@ static Value raw_bytes_to_numeric(GlobalObject& global_object, ByteBuffer raw_va
     }
 }
 
-// 25.1.2.10 GetValueFromBuffer ( arrayBuffer, byteIndex, type, isTypedArray, order [ , isLittleEndian ] ), https://tc39.es/ecma262/#sec-getvaluefrombuffer
+// Implementation for 25.1.2.10 GetValueFromBuffer, used in TypedArray<T>::get_value_from_buffer().
```

Comment:
Odd change, why remove the arguments+link and mention where it's used instead?

Reply:
We had two functions with the `GetValueFromBuffer` comment on them, `get_value_from_buffer()` is the right one - it just happens to wrap this.